site/content/en/docs/Guides/Client SDKs/_index.md

```diff
@@ -316,6 +316,103 @@ or SDK.WatchGameServer() instead to list the connected players.
 
 [playerstatus]: {{< ref "/docs/Reference/agones_crd_api_reference.html#agones.dev/v1.PlayerStatus" >}}
 
+### Counters And Lists
+
+{{< alpha title="Counters And Lists" gate="CountersAndLists" >}}
+
+The `Counters` and `Lists` features in the SDK offer a flexible configuration for tracking various entities like 
+players, rooms, and sessions. However, it's important to note that this feature has been in Alpha stage and disabled
+by default.
+
+{{< alert title="Note" color="info">}}
+There is a potential race condition when count values are set from both the SDK and through the K8s API(Allocation or otherwise),
```

Comment:
```suggestion
There is a potential race condition when count values are set from both the SDK and through the K8s API (Allocation or otherwise),
```